internal/shellgen/scripts.go

```diff
@@ -46,7 +46,10 @@ func WriteScriptsToFiles(devbox devboxer) error {
 
 	// Write all hooks to a file.
 	written := map[string]struct{}{} // set semantics; value is irrelevant
-	pluginHooks, err := plugin.InitHooks(devbox.InstallablePackages(), devbox.ProjectDir())
+	pluginHooks, err := devbox.PluginManager().InitHooks(
+		devbox.InstallablePackages(),
+		devbox.Config().Include,
```

Comment:
nit: should this field be plural: `Includes`?

Reply:
I think this ship has sailed, has been in the `devbox.json` for a while now.

FWIW, I think we chatted about this an decided to use as a verb instead of noun (similar to `import` instead of `imports`).  I don't feel super strongly, but it seems like it would be difficult to change (but I guess we could support both)

Reply:
oh yeah, I do remember chatting about it being a verb...